chatkit-core/src/main/kotlin/com/pusher/chatkit/ChatkitCore.kt

```diff
@@ -0,0 +1,77 @@
+package com.pusher.chatkit
+
+import com.pusher.chatkit.api.createPlatformClientModule
+import com.pusher.chatkit.state.createStoreModule
+import com.pusher.chatkit.users.api.createUserSubscriberModule
+import com.pusher.util.Result
+import com.pusher.util.asSuccess
+import elements.Error
+import org.koin.core.KoinApplication
+import org.koin.core.module.Module
+import org.koin.dsl.koinApplication
+import org.koin.dsl.module
+
+/**
+ * The entry point for JVM apps to connect with the backend service
+ * and retrieve [ChatkitCore] instance for further interaction with the SDK.
+ */
+class ChatkitCoreConnector {
+
+    /**
+     * Creates an object of this class for your Chatkit instance
+     * using passed [TokenProvider][com.pusher.platform.tokenProvider.TokenProvider]
+     * (for authentication and user identification) and additional optional dependencies.
+     *
+     * @param instanceLocator available in the [dashboard](https://dash.pusher.com/chatkit/)
```

Comment:
maybe instead of using the word "available" - use "can be found" - e.g. the instanceLocator can be found in the dashboard?

Reply:
396f6436d68c00efa8b7fbeb05e30fb827d928e7